ScnSoft-Education/LeetCodeTraining/ReverseInteger7/ReverseIntegerSolution.cs

```diff
@@ -10,11 +10,7 @@ public static int Reverse(int x)
             {
                 int digit = x % 10;
                 x /= 10;
-                if (reversedX > int.MaxValue / 10 || (reversedX == int.MaxValue / 10 && digit > 7))
-                {
-                    return 0;
-                }
-                if (reversedX < int.MinValue / 10 || (reversedX == int.MinValue / 10 && digit < -8))
+                if ((reversedX > int.MaxValue / 10 || (reversedX == int.MaxValue / 10 && digit > 7)) || (reversedX < int.MinValue / 10 || (reversedX == int.MinValue / 10 && digit < -8)))
```

Comment:
Please split long lines (more than 120 symbols, in this case it is 170) of the code for better viewing, in this case, similar to this:
```
if ((reversedX > int.MaxValue / 10 || (reversedX == int.MaxValue / 10 && digit > 7))
||
(reversedX < int.MinValue / 10 || (reversedX == int.MinValue / 10 && digit < -8)))
```